modin/experimental/xgboost/xgboost.py

```diff
@@ -99,6 +94,12 @@ def train(
                          'eval': {'logloss': ['0.480385', '0.357756']}}}
     """
     LOGGER.info("Training started")
+
+    if Engine.get() == "Ray":
+        from .xgboost_ray import _train
+    else:
+        raise ValueError("Current version supports only Ray engine as MODIN_ENGINE.")
```

Comment:
https://github.com/modin-project/modin/pull/2643#discussion_r563659173

Reply:
Thanks, it was fixed